src/components/ux-button/ux-button.hbs

```diff
@@ -1,13 +1,30 @@
-<a class="button {{class}}"
-	{{#href}}href="{{href}}"{{/href}}
-	{{#role}} role="{{role}}" {{/role}}
-	{{#ariaLabel}} aria-label="{{ariaLabel}}" {{/ariaLabel}}
-	{{#tabindex}} tabindex="{{tabindex}}" {{/tabindex}}
-	on-tap="clickHandler()"
+{{#if href}}
+	<a class="button {{class}}"
+		{{#if href}}href="{{href}}"{{/if}}
```

Comment:
The else statement uses a button instead of an anchor. It's an accessibility change, anchors aren't valid without a href